Dizzy/Screens/Discovery/DiscoveryVC.swift

```diff
@@ -46,14 +46,15 @@ class DiscoveryVC: ViewController {
         themeImageView.snp.makeConstraints { themeImageView in
             
             themeImageView.top.leading.trailing.equalToSuperview()
-            themeImageView.bottom.equalTo(view.snp.centerY).offset(25)
+            themeImageView.height.equalTo(view.snp.height).multipliedBy(0.5)
         }
         
         nearByPlacesView.snp.makeConstraints { nearByPlacesView in
             
-            nearByPlacesView.top.equalTo(view.snp.centerY)
-            nearByPlacesView.leading.trailing.equalToSuperview()
-            nearByPlacesView.bottom.equalTo(view.safeAreaLayoutGuide.snp.bottom)
+            nearByPlacesView.height.equalTo(view.snp.height).multipliedBy(0.55)
+            nearByPlacesView.leading.equalToSuperview().offset(5)
+            nearByPlacesView.trailing.equalToSuperview().offset(-5)
+            nearByPlacesView.bottom.equalTo(view.safeAreaLayoutGuide.snp.bottom).offset(-5)
```

Comment:
this layout will make the bottom of iphone x and above blank and cause black line to appear at the bottom, if there is not a tab bar you should make this untill bottom and make offset to the collection view. talk with me if you did not understand

Reply:
Done!